driven/storage/adapter.go

```diff
@@ -1203,19 +1203,26 @@ func (sa *Adapter) FindAccountsByParams(searchParams map[string]interface{}, app
 	for i, appOrg := range appOrgs {
 		appOrgIDs[i] = appOrg.ID
 	}
-	searchParams["app_org_id"] = appOrgIDs
-	filter := sa.getFilterForParams(searchParams)
 
-	var accounts []map[string]interface{}
 	options := options.Find()
-	options.SetLimit(int64(limit))
-	options.SetSkip(int64(offset))
+	var filter bson.M
+	if rangeIndex == "" {
+		searchParams["app_org_id"] = appOrgIDs
+		filter = sa.getFilterForParams(searchParams)
+		options.SetLimit(int64(limit))
+		options.SetSkip(int64(offset))
+	} else {
+		filter = bson.M{"_id": bson.M{"$lt": rangeIndex}}
```

Comment:
Since we are sorting the documents in ascending order I think you want to use `$gt` here